apps/payments/next/app/[locale]/[offeringId]/[interval]/upgrade/[cartId]/(startLayout)/start/page.tsx

```diff
@@ -65,98 +66,111 @@ export default async function Upgrade({
-        {l10n.getString(
-          'upgrade-page-acknowledgment',
-          {
-            nextInvoiceDate: l10n.getLocalizedDate(
-              cart.upcomingInvoicePreview.nextInvoiceDate
-            ),
-          },
-          `Your plan will change immediately, and you’ll be charged a prorated
+        <p
+          className="leading-5 text-sm"
+          data-testid="sub-update-acknowledgment"
+        >
+          {l10n.getString(
+            'upgrade-page-acknowledgment',
+            {
+              nextInvoiceDate: l10n.getLocalizedDate(
+                cart.upcomingInvoicePreview.nextInvoiceDate
+              ),
+            },
+            `Your plan will change immediately, and you’ll be charged a prorated
           amount today for the rest of this billing cycle. Starting
           ${l10n.getLocalizedDateString(
             cart.upcomingInvoicePreview.nextInvoiceDate
           )}
           you’ll be charged the full amount.`
-        )}
-      </p>
+          )}
+        </p>
 
-      <div
-        className="border-b border-grey-200 my-6"
-        role="separator"
-        aria-hidden="true"
-      ></div>
+        <div
+          className="border-b border-grey-200 my-6"
+          role="separator"
+          aria-hidden="true"
+        ></div>
 
-      {/*
+        {/*
         If currency could not be determiend, it is most likely due to an invalid
         or undetermined tax address. Future work will add the Tax Location picker
         which should allow a customer to set their tax location, which would then
         provide a valid currency.
       */}
-      {cart.currency &&
-        cart.taxAddress?.countryCode &&
-        cart.taxAddress?.postalCode && (
-          <PaymentSection
-            cmsCommonContent={cms.commonContent}
-            paymentsInfo={{
-              amount: cart.amount,
-              currency: cart.currency.toLowerCase(),
-            }}
-            cart={{
-              ...cart,
-              currency: cart.currency,
-              taxAddress: {
-                countryCode: cart.taxAddress.countryCode,
-                postalCode: cart.taxAddress.postalCode,
-              },
-            }}
-            locale={locale}
-            sessionUid={session?.user?.id}
-          />
-        )}
-    </section>
+        {cart.currency &&
+          cart.taxAddress?.countryCode &&
+          cart.taxAddress?.postalCode && (
+            <PaymentSection
+              cmsCommonContent={cms.commonContent}
+              paymentsInfo={{
+                amount: cart.amount,
+                currency: cart.currency.toLowerCase(),
+              }}
+              cart={{
+                ...cart,
+                currency: cart.currency,
+                taxAddress: {
+                  countryCode: cart.taxAddress.countryCode,
+                  postalCode: cart.taxAddress.postalCode,
+                },
+              }}
```

Comment:
(understanding this came over from before but --) This looks like maybe it can be simplified to just `cart={cart}`, since `currency` and `taxAddress` exist on the cart

Reply:
`TaxAddress` and `currency` used to be optional fields, so it may be a holdover from that